op-node/flags/flags.go

```diff
@@ -45,6 +46,16 @@ var (
 		EnvVars: prefixEnvVars("NETWORK"),
 	}
 	/* Optional Flags */
+	SyncModeFlag = &cli.GenericFlag{
+		Name:    "syncmode",
+		Usage:   "IN DEVELOPMENT: Specify full or snap sync. In reality full=CL sync and snap=EL sync.",
```

Comment:
Its complicated with the naming - is it snap or EL?

Reply:
both. There's some other comments which explain this difficulty more.